src/sidebar/util/time.js

```diff
@@ -195,48 +198,49 @@ function nextFuzzyUpdate(date, now) {
  * update frequency depends on the age of a timestamp.
  *
  * @param {String} date - An ISO 8601 date string timestamp to format.
- * @param {Date} callback - A callback function to call when the timestamp changes.
+ * @param {Function} callback - A callback function to call when the timestamp changes.
  * @return {Function} A function that cancels the automatic refresh.
  */
 function decayingInterval(date, callback) {
   let timer;
   const timeStamp = date ? new Date(date) : null;
-  const update = function() {
+
+  const update = () => {
     const fuzzyUpdate = nextFuzzyUpdate(timeStamp, new Date());
     if (fuzzyUpdate === null) {
       return;
     }
-    const nextUpdate = 1000 * fuzzyUpdate + 500;
-    timer = setTimeout(function() {
+    const nextUpdate = fuzzyUpdate + 500;
+    timer = setTimeout(() => {
```

Comment:
I had tiny concerns about binding here with the arrow function. Sanity check?

Reply:
It should be fine. It only matters if you access `this` within the body of the function. There are some very subtle other differences between arrow functions and anonymous functions ([example](https://github.com/preactjs/enzyme-adapter-preact-pure/pull/39)) but the use of `this` is the only one which matters to most code AFAIK.